src/submission/tests.py

```diff
@@ -609,7 +609,7 @@ def test_article_encoding_ris(self):
             PY  - 1990
             TI  - Test article: A RIS export test case
             T2  - {journal_name}
-            UR  - http://localhost/TST/article/id/{article_id}/
+            UR  - http://testserver/article/id/{article_id}/
```

Comment:
For this change to work consistently, the test needs to run under domain mode explicitely, otherwise the test outcome can change based on user settings. See this for an example:

https://github.com/BirkbeckCTP/janeway/blob/5511797544a27e30751b156474b08da42f401fb6/src/repository/tests/test_views.py#L58